Services/Mail/classes/Setup/class.ilMailDatabaseUpdateSteps.php

```diff
@@ -31,4 +31,40 @@ public function step_1() : void
             $this->db->dropTableColumn('mail', 'm_email');
         }
     }
+
+    public function step_2() : void
+    {
+        $result = $this->db->queryF('SELECT value FROM settings WHERE module = %s AND keyword = %s', ['text', 'text'], ['common', 'mail_system_sys_signature']);
+        $row = $this->db->fetchAssoc($result);
+        if (isset($row['value']) && $row['value'] !== null) {
+            $this->db->replace(
+                'settings',
+                [
+                    'module' => 'common',
+                    'keyword' => 'mail_system_sys_signature'
+                ],
+                [
+                    'value' => str_replace(['[CLIENT_NANE]', '[CLIENT_DESC]', '[CLIENT_URL]'], ['[INSTALLATION_NAME]', '[INSTALLATION_DESC]', '[ILIAS_URL]'], $row['value'])
+                ]
+            );
+        }
+    }
+
+    public function step_3() : void
+    {
+        $result = $this->db->query("SELECT tpl_id, m_message FROM mail_man_tpl WHERE m_message LIKE '%[CLIENT_NAME]%'");
+        while ($row = $this->db->fetchAssoc($result)) {
+            if (isset($row['m_message'], $row['tpl_id']) && $row['m_message'] !== null && $row['tpl_id'] !== null) {
+                $this->db->replace(
+                    'mail_man_tpl',
+                    [
+                        'tpl_id' => (int) $row['tpl_id'],
+                    ],
+                    [
+                        'm_message' => str_replace('[CLIENT_NANE]', '[INSTALLATION_NAME]', $row['m_message'])
+                    ]
+                );
+            }
+        }
+    }
```

Comment:
IMO we need a further step to replace the content in the `body` field of table `mail_template`.